proxy/caddy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+title: Setting up Caddy
+permalink: /docs/proxy/caddy/
+---
+The following configurations works for HTTPS (with an HTTP redirection).
```

Comment:
There's a spelling mistake: "the following configuration works"

Reply:
Fixed in 851966ee361492f90686cc78fd07d65bd65be3d6